getting_started.md

```diff
@@ -0,0 +1,169 @@
+# Getting started
+
+This is a short tutorial that runs you through some basic functionality of the poseidon framework and tooling.
+
+## Preparation
+You will need to install our two command-line tools `trident` and `xerxes`. They are available as pre-compiled binaries for MacOS and Linux, and in case of trident also for Windows: 
+* `trident`: https://github.com/poseidon-framework/poseidon-hs/releases/latest
+* `xerxes`: https://github.com/poseidon-framework/poseidon-analysis-hs/releases/latest
+
+You can also install both tools from bioconda: [trident](https://anaconda.org/bioconda/poseidon-trident), [xerxes](https://anaconda.org/bioconda/poseidon-xerxes)
+
+For the following, we assume you have the executables for your operating system somewhere on your path.
+
+## Getting some remote data
+
+Let's check which packages are available remotely:
+
+```bash
+trident list --remote --packages
+```
+
+You will see a long list of packages. Let's check which packages has French individuals
+
+```bash
+trident list --remote --groups --raw | grep 'French'
+```
+
+Among the packages listed you should find `2012_PattersonGenetics`. 
+Let's download that one, and let's also add three further packages. We will download them into a new directory named `~/poseidon_repo` right in your home folder:
+
+?> Tipp: You can also use `trident list` to view entities in your own local repository. You can list packages, groups or individuals. Check out `trident list --help` or the [documentation](trident.md)
+
+```bash
+mkdir -p ~/poseidon_repo
+trident fetch -f '*2012_PattersonGenetics*,*Reference_Genomes*,*Archaic_Humans*' -d ~/poseidon_repo
+```
+
+## Simple analyses
+
+Let's analyse the newly downloaded data using some F-statistics. Let's show that people with West-Eurasian ancestry are significantly more closely related to Neanderthals than people with West-African ancestry are:
+
+```bash
+xerxes fstats -d ~/poseidon_repo --stat 'F4(<Chimp.REF>,<Altai_published.DG>,Yoruba,French)' \
+  --stat 'F4(<Chimp.REF>,<Altai_published.DG>,Sardinian,French)'
+```
+
+Here is the result:
+```
+.-------------------------------------------------------.-----------------------.-----------------------.--------------------.
+|                       Statistic                       |       Estimate        |        StdErr         |      Z score       |
+:=======================================================:=======================:=======================:====================:
+| F4(<Chimp.REF>,<Altai_published.DG>,Yoruba,French)    | 1.5829336247989434e-3 | 1.9812755791513955e-4 | 7.989467197071763  |
+| F4(<Chimp.REF>,<Altai_published.DG>,Sardinian,French) | 7.610136296417463e-5  | 6.920533028304284e-5  | 1.0996459760097628 |
```

Comment:
I'm a bit spooked by the number of decimal places in `1.5829336247989434e-3`. This is not relevant for this particular PR, but is this a good summary of the F4 result? Wouldn't it be more easy to digest if this number would be rounded? Maybe a switch in the xerxes interface could allow to toggle between rounded and a non-rounded output?

Reply:
OK, one idea is to simply always round them here in the standard output, but keep them full in the table output using `-f`. I have added that as ToDo

Reply:
Done!